src/js/modules/Contact/Phone.js

```diff
@@ -12,4 +12,10 @@ const ContactPhone = ({ className, phone, ttyphone }) => (
   </div>
 );
 
-export default ContactPhone;
+Phone.propTypes = {
+  phone: PropTypes.string.isRequired,
+  className: PropTypes.string,
+  ttyphone: PropTypes.string,
```

Comment:
if ttyphone is not required, we should probably update the JSX to optionally not render the property if it's not set